<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusWindow</class>
 <widget class="QWidget" name="StatusWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="KComboBox" name="nickSelector">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Aki::ChatInput" name="chatInput"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="Aki::ChatView" name="chatOutput" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="Aki::SearchBar" name="channelSearch" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>Aki::ChatInput</class>
   <extends>QLineEdit</extends>
   <header>ui/chatinput.h</header>
  </customwidget>
  <customwidget>
   <class>Aki::ChatView</class>
   <extends>QWidget</extends>
   <header>ui/chatview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Aki::SearchBar</class>
   <extends>QWidget</extends>
   <header>ui/searchbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
